docs/Foundations.md

```diff
@@ -0,0 +1,56 @@
+# Foundations
+
+## Current State
+
+To develop on Xilinx FPGAs, there are a couple of workflows:
+
+1. Legacy HLS -> Vivado workflow
+2. Vitis Kernel workflow (newest)
```

Comment:
Maybe a comparison matrix can be made as a resource which can be useful if this information has to be consulted on a regular basis.